Library/Formula/deheader.rb

```diff
@@ -1,12 +1,16 @@
 class Deheader < Formula
   homepage "http://www.catb.org/~esr/deheader"
-  url "http://www.catb.org/~esr/deheader/deheader-1.0.tar.gz"
-  sha1 "dc9fc816af1631a84ace7f94a85a3a424d72dbed"
-  head "https://git.gitorious.org/deheader/deheader.git"
+  url "http://www.catb.org/~esr/deheader/deheader-1.1.tar.gz"
+  sha256 "69f69e9c7d9398221cb49f7de91df0d122e4b0ec942bede2d7c592401e4b913c"
+  head "git://thyrsus.com/repositories/deheader.git"
+
+  depends_on :python
```

Comment:
Maybe that could be in `brew audit` somewhere, too, unless there is ever a case where an unadorned `depends_on :python` is necessary. Just a thought. :smile_cat: